pennylane/hf/convert.py

```diff
@@ -0,0 +1,316 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
```

Comment:
I suggest moving the `convert.py` module from `pennylane.hf` to `pennylane`. Suggestions for the name of the module are also welcome. (The equivalent of this module for converting circuits is named `io.py`.)